.github/actions/checkov-scans-v1/README.md

```diff
@@ -0,0 +1,36 @@
+# checkov-scans-v1
+
+A GitHub Action to run [Checkov scans](https://github.com/bridgecrewio/checkov-action/tree/v12#readme) and a result will be shown in Prisma Cloud.
+
+## Inputs
+
+| Name                | Required | Description                       | Default |
+| ------------------- | -------- | --------------------------------- | ------- |
+| `output-file-path`  | Yes      | Output file path of Checkov scans | NA      |
```

Comment:
I know Mobile do this, but we shouldn't have to save each scan into some named directory, they are available on Prisma Cloud, seems a bit redundant, thoughts? 

```suggestion
```

Reply:
To be honest I don't know, because I couldn't test it in the worked repo. This variable is used for bridgecrewio/checkov-action@v12. Here is from docs. So when I can test it in the worked repo I will remove this param if it's unnecessary.

![image](https://github.com/dequelabs/axe-api-team-public/assets/160499295/f047d3f7-ef24-4f6c-8d3f-5f2a7b7e850f)

Reply:
Ah ok, ya the action will upload the scan results so no need to save them within the repo IMO

Reply:
deleted the input `output-file-path`. Also pushed the changes of workflow for jazzband, axe-linter